src/platforms/dotnet/common/configuration/msbuild.mdx

```diff
@@ -40,15 +40,14 @@ However, we generally recommend the following:
 
   ```yaml
```

Comment:
should we exclude this yaml?

Reply:
This one is fine I think. It's a github action workflow. No specific spacing requirements here.